pkg/verify/configmap.go

```diff
@@ -8,18 +8,69 @@ import (
 
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/openpgp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
 
 	"github.com/openshift/cluster-version-operator/pkg/verify/store"
 	"github.com/openshift/cluster-version-operator/pkg/verify/store/parallel"
 	"github.com/openshift/cluster-version-operator/pkg/verify/store/sigstore"
+	"github.com/openshift/cluster-version-operator/pkg/verify/verifyutil"
 )
 
-// ReleaseAnnotationConfigMapVerifier is an annotation set on a config map in the
-// release payload to indicate that this config map controls signing for the payload.
-// Only the first config map within the payload should be used, regardless of whether
-// it has data. See NewFromConfigMapData for more.
-const ReleaseAnnotationConfigMapVerifier = "release.openshift.io/verification-config-map"
+const (
+	// ReleaseAnnotationConfigMapVerifier is an annotation set on a config map in the
+	// release payload to indicate that this config map controls signing for the payload.
+	// Only the first config map within the payload should be used, regardless of whether
+	// it has data. See NewFromConfigMapData for more.
+	ReleaseAnnotationConfigMapVerifier = "release.openshift.io/verification-config-map"
+
+	// NamespaceLabelConfigMap is the Namespace label applied to a configmap
+	// containing signatures.
+	NamespaceLabelConfigMap = "openshift-config-managed"
```

Comment:
```console
$ git --no-pager log --oneline -1 origin/pr/413
e792735 (origin/pr/413) Changes to verify package in preparation for move to library-go
$ git --no-pager grep openshift-config-managed origin/pr/413
...some *_test.go hits, where I'm fine sticking with the local string...
origin/pr/413:pkg/verify/store/configmap/configmap.go:		ns:      "openshift-config-managed",
...more places where I'm fine leaving things as they stand...
```

I think we want `ns: importName.NamespaceLabelConfigMap` there.  Or maybe because of import direction we need to turn it around and define the constant in `pkg/verify/store/configmap/configmap.go`?

Reply:
Fixed in [2bde02f](https://github.com/openshift/cluster-version-operator/pull/413/commits/2bde02f1bc058d9b91c9b5cce3afb3259523b74c)